src/CMakeLists.txt

```diff
@@ -211,10 +211,16 @@ set(FLB_DEPS
 if(FLB_SHARED_LIB)
   add_library(fluent-bit-shared SHARED ${src})
   add_sanitizers(fluent-bit-shared)
-  target_link_libraries(fluent-bit-shared ${FLB_DEPS} -lpthread)
   set_target_properties(fluent-bit-shared
     PROPERTIES OUTPUT_NAME fluent-bit)
 
+  # Windows doesn't provide pthread (see winpthreads.c in mk_core).
+  if(MSVC)
```

Comment:
`if(CMAKE_SYSTEM_NAME MATCHES "Windows")` perhaps.

Reply:
No, MinGW provides pthread extension for Windows, but MSVC doesn't.